lib/greenhouse_api/client.rb

```diff
@@ -59,13 +59,13 @@ def list_many(resource, params = {})
           http_method: :get,
           headers: headers,
           endpoint: resource,
-          params: params.merge(page: page, per_page: per_page)
+          params: params.merge(page: page, per_page: params[:per_page] || per_page)
```

Comment:
should we compute `per_page` on line 56 where we're already applying some logic there?

Reply:
great callout, we want to use `per_page` over `limit` and I originally had it like this

```ruby
per_page = params[:per_page] || (limit ? [limit - data.length, MAX_PER_PAGE].min : MAX_PER_PAGE)
``` 

But I think it's important to keep the computation in one place so I'll try a multiline approach.

Reply:
nice - new code looks clean